common/pom.xml

```diff
@@ -26,6 +26,10 @@
 	</parent>
 
 	<artifactId>fhir-common</artifactId>
+	
+	<properties>
+	  <c3p0.version>0.9.5.4</c3p0.version>
```

Comment:
[This comment](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/pull/86#discussion_r583406073) re. indents is not fully resolved yet. Please make sure your indent setting is consistent with the rest of the file. In particular it seems you are using spaces here while the rest of the file has tabs.